src/API/server.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
```

Comment:
I think we need a better name for the express server. Right now we have a `server.js` which is not a descriptive name and could be confusing. We also have a folder called `apiRoutes` that is also confusing because there are many APIs that we are exposing/interacting with. Overall, the terms API and server are very overloaded in this codebase.

Here are a couple names I thought of for the express server that I think could work:

- TaskQueue
- AnswerQueue
- AnswerDispatcher
- QueryDispatcher
- ReasonerQueryDispatcher

I like ReasonerQueryDispatcher the best. It's long, but at least it says exactly what is happening. If you want to go with that we should rename API/server.js -> API/reasonerQueryDispatcher.js and apiRoutes/ -> reasonerQueryDispatcherServer/. API calls would look something like:

```
API.reasonerQueryDispatcher.getAnswer(questionId)
```

Reply:
I don't think we necessarily need Reasoner in the name. But I think QueryDispatcher is a very good descriptive name.